utilities/ltftUtilities.ts

```diff
@@ -388,3 +389,12 @@ export function getStatusReasonLabel(
 
   return "";
 }
+
+export function isValidProgramme(progId: string): boolean {
+  if (!progId) return false;
+  const progIdsToCheckAgainst = store.getState().user.features.ltftProgrammes;
+  if (progIdsToCheckAgainst.length < 1) {
+    return false;
+  }
+  return progIdsToCheckAgainst.some(id => id === progId);
```

Comment:
Assuming you are happy there will be no inconsistent capitalisation issues.

Reply:
Yeah, did have the lower case method when (incorrectly) using prog name but thought UUID was case sensitive? 🤷‍♂️